### PR TITLE
BL-1298 styling changes.

### PR DESCRIPTION
**Pull request by dkinzer**
* Update cite and sendto styling to match other buttons on bookmarks and show page.

### CODE REVIEW

File: app/assets/stylesheets/partials/_header.scss [hunk 1]
```diff
@@ -263,10 +263,10 @@ a#bookmarks_nav {
 
 #sendto-menu a.dropdown-toggle{
  	background-color: $white;
-  border: 0.0625rem solid $header-grey;
+  border: 0.0625rem solid $dark-grey;
```
Comment (sensei100) [inline app/assets/stylesheets/partials/_header.scss:266]: @dkinzer Several of the indentations on this file don't look right.  Are they displaying correctly on your local?

File: app/assets/stylesheets/partials/_vars.scss [hunk 2]
```diff
@@ -12,7 +12,7 @@ $facet-limit: #DDD;
 $grey: #333;
 $light-grey: #f4f6f7;
 $med-grey: #444;
-$dark-grey: #666;
```
Comment (sensei100) [inline app/assets/stylesheets/partials/_vars.scss:15]: Did you check that this variable is not being used other places?  I don't want to inadvertently
change colors that we aren't supposed to.

File: config/locales/en.yml [hunk 3]
```diff
@@ -111,7 +111,7 @@ en:
       sms: "Text"
       refworks: Refworks
       cite_html: "Cite %{current_range}"
-
+      sendto: "Send To %{current_range}"
```
Comment (sensei100) [inline config/locales/en.yml:114]: Will adding this here also display ${current_range} on the individual record pages?

File: app/assets/stylesheets/partials/_vars.scss (hunk 2 above)
Comment (dkinzer) [reply app/assets/stylesheets/partials/_vars.scss:15]: I thought I updated that variable... maybe I forgot to commit....

Comment (sensei100) [reply app/assets/stylesheets/partials/_vars.scss:15]: You did update it. My concern is whether other elements are using that variable that will also be changed.

File: config/locales/en.yml (hunk 3 above)
Comment (dkinzer) [reply config/locales/en.yml:114]: No, the individual record page uses it's own template and does not use this local 

image

File: app/assets/stylesheets/partials/_vars.scss (hunk 2 above)
Comment (dkinzer) [reply app/assets/stylesheets/partials/_vars.scss:15]: as far as I know this was being used for the grey around the buttons.

File: config/locales/en.yml (hunk 3 above)
Comment (sensei100) [reply config/locales/en.yml:114]: I can see that it is also used as a filter-separator in the facets.  Not sure about anywhere else.  I'm just not sure we should change a variable when it is used in multiple places.

File: app/assets/stylesheets/partials/_vars.scss (hunk 2 above)
Comment (sensei100) [reply app/assets/stylesheets/partials/_vars.scss:15]: I can see that it is also used as a filter-separator in the facets.  Not sure about anywhere else.  I'm just not sure we should change a variable when it is used in multiple places unless we know that they want everything updated to the new color.

File: app/assets/stylesheets/partials/_header.scss (hunk 1 above)
Comment (dkinzer) [reply app/assets/stylesheets/partials/_header.scss:266]: Yeah they were looking right on my local.

image
image

File: app/assets/stylesheets/partials/_vars.scss (hunk 2 above)
Comment (dkinzer) [reply app/assets/stylesheets/partials/_vars.scss:15]: supposedly facets use it... but I can't see much of a difference.

**Review by sensei100 — APPROVED** [review]

**Pull request merged**